Rakefile

```diff
@@ -6,18 +6,23 @@ Rake.application.load_imports
 
 task('metrics:mutant').clear
 namespace :metrics do
-  task :mutant => :coverage do
-    success = Kernel.system(*%w[
+  task mutant: :coverage do
+    jobs = 4 if ENV.key?('CIRCLE_CI')
+
+    arguments = %w[
       bundle exec mutant
-      --zombie
-      --use rspec
+      --ignore-subject Mutant::Meta*
       --include lib
-      --require mutant
       --since HEAD~1
-      --ignore-subject Mutant::Meta*
-      --
-      Mutant*
-    ]) or fail 'Mutant task is not successful'
+      --require mutant
+      --use rspec
+      --zombie
+    ]
+    arguments.concat(%W[--jobs 4]) if ENV.key?('CIRCLE_CI')
```

Comment:
I do not like this inline signalling. Assinging `0` a special behavior seems to be weird. If you right now boot mutant with `--jobs 0` the system behaves like you'd expect: Boots but does not make progress, as no workers where spawned.